### DISCUSSION TITLE
Catalina

### DISCUSSION THREAD

**Discussion by trayyan** [vovinacci/OpenCore-ASUS-ROG-MAXIMUS-XI-HERO#99] [Q&A] [answered]
Good job and thanks. I have two questions

1. Can I use your latest EFI folder with version Catalina?
2. Also, I tried to edit config.plist with propertree and it shows an error "Incorrect padding" I have installed python3 on my MacBook and tried from windows as well. Any idea why? Can I add smbios details with a text editor?

**Comment by vovinacci** [comment]
@trayyan My apologies, I've missed this completely, so better late than never.

- I don't have Catalina anymore, however, according to OpenCore documentation it should just work
- It's hard for me to tell what went wrong, however just editing plist file is fine. I do it all the time :) As an option, you may want to run `plutil -lint <file>` to check what went wrong it attach it here and I'll take a look. This utility is shipped with macOS, so don't need to install anything.

**Reply by trayyan** [reply]
Thanks. I am planning to upgrade my gpu to 6600xt,
Will the latest efi folder support macOS Monterey and this GPU?

**Reply by vovinacci** [reply]
- Monterey is supported for sure.
- Not sure about GPU - I don't have it, cannot be that sure. Looking at the various forums and OpenCore documentation - should work out the box.
- Looking at [this thread](https://www.reddit.com/r/hackintosh/comments/sm7xzp/navi_23_rx_6600_xt_hardware_acceleration_in/), you would need to upgrade to Monterey first and then use it.

I would be very much curious to hear a feedback from you if it works.

**Comment by trayyan** [comment]
The main issue I am facing is now, I cant update the config.plist file with
Propertree app. It shows incorrect padding, what should I do?

**Reply by vovinacci** [reply]
As I understand, all you need is to replace placeholders with serial number and so on, correct? 
If that's the case, I can try to help you. You may want to send `config.plist` after the edit, I'll check it for you.

**Comment by trayyan** [comment]
I have installed Monterey amd it’s working, i have problems with USB. The ports you mentioned it is not working and internal usb not working at all. My Bluetooth is connected to thats. Any idea to fix?

**Reply by vovinacci** [reply]
Hmmm, USB should work as pictured. I had it confirmed by various people.

You may want to follow https://dortania.github.io/OpenCore-Post-Install/usb/intel-mapping/intel.html, map the same ports as on the [picture](https://github.com/vovinacci/OpenCore-ASUS-ROG-MAXIMUS-XI-HERO/blob/master/assets/usb-mapping.png) and check the difference between newly generated and existing `USBMap.kext`. I am also curious to see the difference.

**Comment by vovinacci** [comment]
@trayyan Did it work for you in the end?

**Comment by trayyan** [comment]
I have issues with USB. Front port USB not working in 2.0. And some of back
port same issue. In the USB Mapping image, you mentioned you put two-port
will not work at all, but for me, it is working as 2.0
Graphic card is not tested yet, I was gonna sell my old rx580 but no one is
buying at the moment because of the crypto crash

**Comment by vovinacci** [comment, marked as answer]
@trayyan I'm closing this conversation as there's nothing really I can help with as of now. Feel free to start a new discussion once. With regards to USB, still curious to see what would be the difference as per https://github.com/vovinacci/OpenCore-ASUS-ROG-MAXIMUS-XI-HERO/discussions/99#discussioncomment-2762649.

**Comment by trayyan** [comment]
I give up. I sold everything. Thanks for your help

**Reply by vovinacci** [reply]
These are sad news. However, life goes on. Take care!
